### PR TITLE
utils_test.libvirt: add function create_channel_xml

### PR DESCRIPTION
**Pull request by PandaWei**
Extract the common that used to create channel xml file
in tp-libvirt, and regard it as a common funciton

### CODE REVIEW

**Comment by yumingfei** [comment]
It's good, but please update it for conflicts.

**Comment by PandaWei** [comment]
@yumingfei 
Hi, Thanks for your reminder.
I have updated it.

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -1668,6 +1669,54 @@ def create_nwfilter_xml(params):
...
+    channel_params = {}
+    try:
+        channel_type_name = params.get("channel_type_name")
+        source_mode = params.get("source_mode", "")
+        source_path = params.get("source_path", "")
+        target_type = params.get("target_type", "")
+        target_name = params.get("target_name", "")
+    except Exception, detail:
+        logging.error("Fail to create channel XML:\n%s", detail)
+
```
Comment (will-Do) [inline virttest/utils_test/libvirt.py]: I don't think it will raise any exception here, maybe your want to check these variables are not ```None/""``` ?

Comment (PandaWei) [inline virttest/utils_test/libvirt.py]: yes， it's what i want to do。
do you have any suggestion?

**Comment by PandaWei** [comment]
@will-Do 
I updated it .
May i trouble you to check it again?

File: virttest/utils_test/libvirt.py [hunk 2]
```diff
@@ -1668,6 +1669,55 @@ def create_nwfilter_xml(params):
...
+    channel_alias = {}
+    channel_address = {}
+    channel_params = {}
+
+    channel_type_name = params.get("channel_type_name", None)
+    source_mode = params.get("source_mode", None)
+    source_path = params.get("source_path", None)
+    target_type = params.get("target_type", None)
+    target_name = params.get("target_name", None)
+
```
Comment (will-Do) [inline virttest/utils_test/libvirt.py]: The default ```None``` value could be omitted, any way, it's okay for me.
ACKed.

Comment (yumingfei) [inline virttest/utils_test/libvirt.py]: Yes, it's weird to add useless None here. :)

Comment (PandaWei) [inline virttest/utils_test/libvirt.py]: @yumingfei 
deleted 'None'.

**Comment by will-Do** [comment]
Okay, merger this PR, thanks

**Pull request merged**